server/security/index.js

```diff
@@ -30,6 +30,13 @@ exports.applyAuthMiddleware = (keycloakConfig, expressRouter, apiPath) => {
     // Protect the main route for all graphql services
     // Disable unauthenticated access
     expressRouter.use(apiPath, keycloak.protect())
+
+    expressRouter.get('/login', keycloak.protect(), function (req, res) {
+      const token = JSON.stringify(JSON.parse(req.session['keycloak-token']))
+      res.send({
```

Comment:
If you change this to the following, it's a little bit handier.

```js
const token = JSON.parse(req.session['keycloak-token'])
res.json({ authorization: `Bearer ${token.access_token}`})
```

Reply:
Yeah. done that already. Just haven't pushed :D